### ISSUE TITLE
Cannot integrate Charts to VueJS SPA (babel/webpack)

### ISSUE THREAD

**Issue by yuriygeorge** [tradingview/charting-library-examples#122]
In examples for VueJS using '@vue/cli', it all works fine, but with babel/webpack (with my project integration), I have got some problems with imports.

Our vue.js project doesn't use the '/public' folder, as it does in the vue.js tradingview examples. So, instead of putting the files and folders into '/public' I place them into '/static' 
 folder instead , and also changing the script imports.

I cannot find imports for '/public/charting_library' folder, but I suppose this imports may be hardcoded in lib code?

First error I got: "export 'widget' was not found in './charting_library/charting_library.min" 

I  solved it by adding:

["transform-es2015-modules-commonjs", {
"allowTopLevelThis": true
}],

to babel plugins in .babelrc

Other problems after fixes:

Uncaught SyntaxError: Unexpected token < polyfills.js:1
Uncaught SyntaxError: Unexpected token < bundle.js:1
Uncaught (in promise) TypeError: t._innerWindow(...).widgetReady is not a function at eval (charting_library.min.js?4897:1) charting_library.min.js?4897:1

Thanks for any help.

TLDR; the vue.js examples use vue-cli to run the (npm run serve) command. But, we are using webpack and can't integrate the charting library due to issues. Is the "/public" folder hardcoded, or can this be configured in a configuration file, say to use a different folder, such as "/static" instead.

**Comment by skywalker2013** [comment]
@yuriygeorge how did you solve issue ?
